src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -203,10 +254,19 @@ export class FacetSlider extends Component {
         })
       }
     }),
+
     /**
-     * Specifies a function that will generate the steps for the slider. The function receives the slider boundaries and must return an array of number (the steps).
+     * Specifies a function to generate the steps for the FacetSlider (see {@link FacetSlider.options.steps}. This
+     * function receives the FacetSlider boundaries (see {@link FacetSlider.options.start} and
+     * {@link FacetSlider.options.end}) and must return an array of number (the steps).
```

Comment:
I discussed it with François Dallaire and he agrees with your idea of using plural when talking about multiple items of the same type, so I'm going to do this after all :)